git-cheatsheet.md

```diff
@@ -10,3 +10,10 @@ Git branch commands
 | `git branch`                 | list your branches                   |
 | `git branch -a`              | list all branches (local and remote) |
 | `git branch -d <branchname>` | delete a branch                      |
+
+| Git command               | Git task                                         |
+| ------------------------- | ------------------------------------------------ |
+| `git status`              | List all files that have changed and their state |
+| `git add <filename>`      | Add a file to the stage                          |
+| `git commit -m "add foo"` | Create a commit including all staged files       |
+| `git log --oneline`       | Show the commit history                          |
```

Comment:
| `git log --oneline`       | Show the commit history in only one line                          |

Reply:
you could also add this row:

| git log | Show complete commit history |